src/styles/components/_form.scss

```diff
@@ -36,22 +33,45 @@
   &__input-label {
     display: block;
     margin-bottom: rem(12px);
+
+    font-size: rem(16px);
+    font-weight: 600;
+    line-height: 1.2;
+
+    @include respond-above('lg') {
+      font-size: rem(20px);
+    }
   }
 
   &__input-group {
-    width: 100%;
+    &:not(:last-child) {
+      margin-bottom: rem(12px);
+
+      @include respond-above('md') {
+        margin-bottom: rem(32px);
+      }
+    }
   }
 
-  &__actionable,
-  &__recaptcha {
-    display: flex;
-    justify-content: center;
+  &__input-password {
+    position: relative;
   }
 
   &__input {
-    width: 100%;
+    display: block;
```

Comment:
In this case, would be better to use `div` on `form__help` and remove this property.

Reply:
Did you mean to change `<span>` into `<div>` ? If yes, I think this is not an approach that I followed. 

![image](https://user-images.githubusercontent.com/32285869/112720173-53f91600-8f2f-11eb-9ce9-2cb2e6cddc76.png)

If no, I'm understand in the way to use `span` as usual and delete this property in `&__input` and add `display: block` in `&__help`. 🙏🏻

Reply:
This is a guide which we can follow or not. The naming convention is great, but our case if we set `display: block` at input is not sufficient. Setting `display: block` at `span` isn't bad, but then this is just a simple `div` behavior. So why not just use `div`?

Reply:
Ok, I understand your point now. Thanks for your guide.  🙏🏻